packages/postgres-database/src/simcore_postgres_database/models/_common.py

```diff
@@ -34,8 +43,8 @@ def column_created_by_user(
         sa.Integer,
         sa.ForeignKey(
             users_table.c.id,
-            onupdate="CASCADE",
-            ondelete="SET NULL",
+            onupdate=ReferentialAction.CASCADE,
```

Comment:
minor: is this not already a literal? not sure if it makes sense to create a class for sqlalchemy types.

Reply:
I also do not see the point of defining this now. I'd guess that in other places this object is not used. I would avoid it if possible.

Reply:
For example, I see value in it. I never know what the options are, so this would be handy.

Reply:
@sanderegg @GitHK this was a recommendation by sonar and I think it makes total sense: having repeated strings is better to define a constant. In addition, as @matusdrobuliak66 says, it allows you to see which options you have (I even left a link to the docs to see what they mean)
@sanderegg I use the class as scope. note that these are **class attributes** , but I could also do constants
`REFERENTIAL_ACTION__CASCADE` etc

From this point on, I would use it everywhere in the definitions of foreign keys within the postgres models. I did not want to pollute this PR with more changes so I did not do it.